index.js

```diff
@@ -477,14 +477,19 @@ exports.resolveFunctions = function (funcSet) {
  * @param {Array} imports the imports for the module's import table
  * @return {string}
  */
-exports.buildModule = function (funcs, imports = [], exports = [], cbs = []) {
+exports.buildModule = function (funcs, imports = [], callbacks = []) {
   let funcStr = ''
   for (let func of funcs) {
     funcStr += func
   }
-  for (let exprt of exports) {
-    funcStr += `(export "${exprt}" (func $${exprt}))`
-  }
+
+  let callbackTableStr = `
```

Comment:
Wonder if this works if `callback` has no elements. Do you want to make it conditional on that?